packages/qdrant-loader/conf/.env.template

```
@@ -2,11 +2,52 @@
 # Never commit .env with real secrets to version control.
 # Ensure .env is listed in .gitignore (it is by default).
 
-# OpenAI Configuration
-OPENAI_API_KEY=your_openai_api_key_here
+# QDrant Configuration
+QDRANT_URL=http://localhost:6333
+QDRANT_COLLECTION_NAME=my_docs
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The QDRANT_COLLECTION_NAME key should go before the QDRANT_URL key

(UnorderedKey)